docs/web/authentication.md

```diff
@@ -268,7 +273,8 @@ servers as it can elongate the authentication process.
       "groupNameAttr" : "sAMAccountName"
     },
     {
-      "connection_url" : "ldaps://secure.internal.example.org:636",
+      "connection_url"   : "ldaps://secure.internal.example.org:636",
+      "tls_require_cert" : "never",
```

Comment:
You are right. The default behavior shouldn't be insecure. So I think we need to change this value to `null˙ here yes?